static/js/popover_menus.js

```diff
@@ -185,4 +202,140 @@ export function initialize() {
             compose_enter_sends_popover_displayed = false;
         },
     });
+
+    delegate("body", {
+        ...default_popover_props,
+        // The is our minimum supported width for mobile. We shouldn't
+        // make the popover wider than this.
+        maxWidth: "320px",
+        target: ".actions_hover .zulip-icon-ellipsis-v-solid",
+        placement: "bottom",
+        popperOptions: {
+            modifiers: [
+                {
+                    // The placement is set to bottom, but if that placement does not fit,
+                    // the opposite top placement will be used.
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: ["top"],
+                    },
+                },
+            ],
+        },
+        onShow(instance) {
+            on_show_prep(instance);
+            const $row = $(instance.reference).closest(".message_row");
+            const message_id = rows.id($row);
+            const args = popover_menus_data.get_actions_popover_content_args(message_id);
+            instance.setContent(parse_html(render_actions_popover_content(args)));
```

Comment:
Hmm, can you explain why we need a `parse_html` step here? Maybe belongs in a comment.